### PR TITLE
Websocket docker

### PR DESCRIPTION
**Pull request by TonyGiorgio**
TODO:

- [x] Deploy docker images somewhere

### CODE REVIEW

File: websocket-tcp-proxy/src/main.rs [hunk 1]
```diff
@@ -16,19 +17,24 @@ use tower_http::trace::{DefaultMakeSpan, TraceLayer};
...
+    println!("Running websocket-tcp-proxy");
     tracing_subscriber::fmt::init();
 
     let app = Router::new().route("/v1/:ip/:port", get(ws_handler)).layer(
         TraceLayer::new_for_http().make_span_with(DefaultMakeSpan::default().include_headers(true)),
     );
 
-    // TODO let this be configurable
-    let addr = SocketAddr::from(([127, 0, 0, 1], 3001));
+    let port = match env::var("PORT") {
```
Comment (benthecarman) [inline websocket-tcp-proxy/src/main.rs:27]: should make this something like `MUTINY_PROXY_PORT` so it doesn't accidentally overlap with something else

Comment (TonyGiorgio) [reply websocket-tcp-proxy/src/main.rs]: good idea

File: README.md [hunk 2]
```diff
@@ -110,3 +110,16 @@ serve -s build
...
+
+Build the websocket-tcp-proxy image
+
+```
+DOCKER_BUILDKIT=1 docker build -f Dockerfile-Proxy -t bitcoindevshop/websocket-tcp-proxy .
+```
+
+Run the docker image locally
+```
+docker run -d -p 8080:8080 bitcoindevshop/websocket-tcp-proxy
```
Comment (benthecarman) [inline README.md:124]: instead of 8080 should use 3001 as that is the default port

**Comment by TonyGiorgio** [comment]
The proxy is at: `wss://websocket-tcp-proxy-fywbx.ondigitalocean.app`, hopefully soon is at `wss://proxy.mutinywallet.com`

You can put that in to the proxy box. If you want to expose a local port so you can connect to, you can run [bore](https://github.com/ekzhang/bore): `bore local {your_local_port} --to bore.pub` and note the port you get.

Then that connection string would be something like `{pubkey}@159.223.171.199:{bore_port}` - just replace the port with the one you got from bore

**Review by benthecarman — APPROVED** [review]

**Review by futurepaul — APPROVED** [review]
works!

**Pull request merged**